Sources/ElasticSwift/Responses/Response.swift

```diff
@@ -139,7 +139,9 @@ public class Shards: Codable {
 
 public class Hits<T: Codable>: Response {
     
-    public var total: Total?
```

Comment:
At the moment it's the only real compatibility issue I've found to support ES 7.
We need a way to manage this... I'm thinking ...